js/services/Vocabulary.js

```diff
@@ -55,32 +61,19 @@ define(function (require, exports) {
 			searchResultIdentifiers.push(results[c].CONCEPT_ID);
 		}
 		return sourceKey ?
-			CDMResultAPI.getConceptRecordCount(sourceKey, searchResultIdentifiers, results, false) :
-			CDMResultAPI.getConceptRecordCountWithResultsUrl(sharedState.resultsUrl(), searchResultIdentifiers, results, false);
+			CDMResultAPI.getConceptRecordCount(sourceKey, searchResultIdentifiers, results, false, formatter) :
+			CDMResultAPI.getConceptRecordCountWithResultsUrl(sharedState.resultsUrl(), searchResultIdentifiers, results, false, formatter);
 	}
 
-	function search(searchString, options) {
+	async function search(params) {
```

Comment:
no point put 'async' here without 'await' in the body of the function

Reply:
I think there is because async means that we're returning a promise....right?

Reply:
So, let me clarify:   it's not required to define async when you don't use await, however async does indicate that a promise is returned, so I say we leave it in because it's more informative that the function is an async function vs. not knowing if it would return a promise or not.

Reply:
async is simply ignored by the JS interpreter, if there's no await. 
As a developer, when I see async, I expect await in the function, so if there's none it's confusing. So I suggest to keep the code clean.

Reply:
This is not true, and the only thing about async that you should expect is that the function returns a promise. It's not true that the JS interpreter just 'ignores' async when there is no await:  if the function doesn't return a promise, then the function will wrap a promise around the 'raw' value so that the async function returns a promise.

See this for details: https://javascript.info/async-await.

Because this function returns a promise, and I want developers to know that this function returns a promise, I'm leaving this function as 'async'.

Reply:
Yes, sorry, you are right.